### PR TITLE
Add Drone php8.0 pipelines for MariaDB10.4, MySQL8, PGSQL13

### PR DESCRIPTION
**Pull request by solracsf**
Add Drone php8.0 pipelines for MariaDB10.4, MySQL8, PGSQL13

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1: and CI is green minus the usual random fail

**Comment by nickvergessen** [comment]
With 8.1 coming up, I think it's more likely the time to reduce the pipeline.

How likely is it that something works on mariaDB/mysql with 3 different php versions but not with 1 other?

So my suggestion is:

DB | DB version | PHP version | Future
---|---|---|---
MariaDB | 10.1 | 7.3
MariaDB | 10.4 | 7.4 | ❌ 
MariaDB | 10.4 | 8.0
MariaDB | 10.4 | 8.1 | 🔜  don't add ❌
MySQL | 8.0 | 7.4
MySQL | 8.0 | 8.0 | ❌ 
MySQL |  8.0 | 8.1 | 🔜 
Postgres | 9.6 | 7.3
Postgres | 11 | 7.4
Postgres | 12 | 7.4 | ❌ 
Postgres | 13 | 7.4 | ❌ 
Postgres | 13 | 8.0
Postgres | 13 | 8.1 | 🔜 

This way Postgres is tested on each PHP version once,
MySQL/MariaDB are tested on either PHP version respectively.
Should be good enough, wihtout growing the table (and therefor time) over the top

**Comment by acsfer** [comment]
@nickvergessen tell me what you think.

File: .drone.yml [hunk 1]
```diff
@@ -2186,133 +1932,3 @@ trigger:
...
-    - sh -c "if [ '$DRONE_BUILD_EVENT' = 'pull_request' ]; then bash codecov.sh -B $DRONE_BRANCH -C $DRONE_COMMIT -P $DRONE_PULL_REQUEST -t 117641e2-a9e8-4b7b-984b-ae872d9b05f5 -f tests/autotest-clover-sqlite.xml; fi"
-    - sh -c "if [ '$DRONE_BUILD_EVENT' != 'pull_request' ]; then bash codecov.sh -B $DRONE_BRANCH -C $DRONE_COMMIT -t 117641e2-a9e8-4b7b-984b-ae872d9b05f5 -f tests/autotest-clover-sqlite.xml; fi"
-
-trigger:
-  branch:
-    - master
-    - stable*
-  event:
-    - pull_request
-    - push
```
Comment (nickvergessen) [inline .drone.yml:2287]: opsy?

Comment (acsfer) [reply .drone.yml]: 🙈

**Review by nickvergessen — APPROVED** [review]

**Comment by acsfer** [comment]
@skjnldsv backport?

**Pull request merged**